R/utils-get_code_dependency.R

```diff
@@ -108,25 +110,24 @@ find_call <- function(call_pd, text) {
 #' @noRd
 extract_calls <- function(pd) {
   calls <- lapply(
-    pd[pd$parent == 0, "id"],
+    pd[pd$parent == 0 & pd$token != "COMMENT", "id"],
```

Comment:
I think it also removes comments with `# @linksto` tag, that span to the new line.

```r
tcode <- c("expr <- quote(x <- x + 1)", "x <- 0", "eval(expr) #@linksto x")
parsed_code <- parse(text = tcode, keep.source = TRUE)
pd <- utils::getParseData(parsed_code)
pd <- normalize_pd(pd)

```

<img width="428" alt="image" src="https://github.com/user-attachments/assets/5d0f0f53-e6e1-4c15-8bda-2b58b6d8f933">

Reply:
Brought back in here 0afef8388f8804df5d6006078edd650bb8bee52a